app/javascript/packs/shared/data_table/DataTable.jsx

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import { BootstrapTable, TableHeaderColumn } from 'react-bootstrap-table';
+
+class DataTable extends Component {
+  static cleanColumnAttributes(col) {
+    // Remove additional attributes from the columns
+    const {
```

Comment:
I want to create a hash/map without our custom keys&values (e.g. props). So we can pass this object safely into `<TableHeaderColumn>`, without worrying it will break something. I think this is the simplest way to do it (and, allegedly, cloning `col` and then calling e.g. `delete col.id` is considered slow).